web/themes/custom/usecaselabs/templates/layout/page--404.html.twig

```diff
@@ -0,0 +1,98 @@
+ *
+ * Site identity:
+ * - front_page: The URL of the front page. Use this instead of base_path when
+ *   linking to the front page. This includes the language domain or prefix.
+ *
+ * Page content (in order of occurrence in the default page.html.twig):
+ * - node: Fully loaded node, if there is an automatically-loaded node
+ *   associated with the page and the node ID is the second argument in the
+ *   page's path (e.g. node/12345 and node/12345/revisions, but not
+ *   comment/reply/12345).
+ *
+ * Regions:
+ * - page.header: Items for the header region.
+ * - page.highlighted: Items for the highlighted region.
+ * - page.primary_menu: Items for the primary menu region.
+ * - page.secondary_menu: Items for the secondary menu region.
+ * - page.content: The main content of the current page.
+ * - page.sidebar: Items for the first sidebar.
+ * - page.featured_bottom_first: Items for the first featured bottom region.
+ * - page.featured_bottom_second: Items for the second featured bottom region.
+ * - page.footer_first: Items for the first footer column.
+ * - page.footer_second: Items for the second footer column.
+ * - page.breadcrumb: Items for the breadcrumb region.
+ * - page.social: Items for the social region.
+ *
+ * @see template_preprocess_page()
+ * @see html.html.twig
+ */
+#}
+
+<div id="page-wrapper" class="page-wrapper page-404-error-wrapper">
+	<div id="page">
+
+		{% if page.primary_header or page.secondary_header %}
+			<header id="header" class="site-header" data-drupal-selector="site-header" role="banner">
+
+				<div id="site-header__inner" class="site-header__inner" data-drupal-selector="site-header-inner">
+					<div class="container site-header__inner__container">
+
+						{{ page.primary_header }}
+
+						{% if page.secondary_header %}
+							<div class="mobile-buttons" data-drupal-selector="mobile-buttons">
+								<button class="mobile-nav-button" data-drupal-selector="mobile-nav-button" aria-label="{{ 'Main Menu'|t }}" aria-controls="header-nav" aria-expanded="false">
+									<span class="mobile-nav-button__label">{{ 'Menu'|t }}</span>
+									<span class="mobile-nav-button__icon"></span>
+								</button>
+							</div>
+							<div id="header-nav" class="header-nav" data-drupal-selector="header-nav">
+								{{ page.secondary_header }}
+							</div>
+						{% endif %}
+					</div>
+				</div>
+
+			</header>
+		{% endif %}
+
+		<div id="main-wrapper" class="layout-main-wrapper layout-container">
+			<div id="main" class="layout-main">
+				<div class="main-content">
+					<a id="main-content" tabindex="-1"></a>
+					<div class="main-content__container container">
+            <main role="main">
+              {{ page.content }}
+            </main>
+					</div>
+				</div>
+			</div>
+		</div>
+
+		<footer class="site-footer">
+			<div class="site-footer__inner container">
+				{{ page.footer }}
+			</div>
+		</footer>
+
+		<div class="overlay" data-drupal-selector="overlay"></div>
+	</div>
+</div>
```

Comment:
@shipra-14 we can clean up page twig for 404 page if some region is not required. There could be a case some blocks will be added to the region and they start appearing.